modules/nf-core/agat/spmergeannotations/main.nf

```diff
@@ -0,0 +1,53 @@
+process AGAT_SPMERGEANNOTATIONS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/agat:1.4.0--pl5321hdfd78af_0':
+        'biocontainers/agat:1.4.0--pl5321hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(gffs)
+    path config
+
+    output:
+    tuple val(meta), path("*.gff")  , emit: gff
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args            = task.ext.args ?: ''
+    def prefix          = task.ext.prefix ?: "${meta.id}"
+    def config_param    = config ? "--config $config" : ''
+    def gff_param       = "$gffs".split(' ').collect { "--gff $it" }.join(' ')
```

Comment:
Does this need to be `"$gffs"` or would `gffs` work here?
```suggestion
    def gff_param       = gffs.split(' ').collect { "--gff $it" }.join(' ')
```

Reply:
No, it does not seem to work.